src/resources/auth/middleware/Permit.ts

```diff
@@ -1,18 +1,77 @@
 import express from "express";
-import { User } from "../../../generated/models/User.generated";
 import { PermissionFlag } from "./PermissionFlag";
+import { checkPermissionForRole } from "./Roles";
+import { AuthUser } from "../../../generated/models/AuthUser.generated";
+import { PermissionFilter } from "../filter/PermissionFilter";
+import { ResourcePermissionController } from "../controllers/ResourcePermissionController";
 
 export class Permit {
+	static authorizesToManipulateResource =
+		(resourceController: ResourcePermissionController) =>
+		(req: express.Request, res: express.Response, next: express.NextFunction) => {
+			const identifier = req.params.identifier;
+			const authUser = req.user as AuthUser;
+
+			if (!req.user || !identifier || !authUser || !authUser.organizationIdentifier) {
```

Comment:
Hier ist ein bisschen Redundanz am Start, weil `req.user` und `authUser` ja den gleichen Inhalt haben. Könnte man also etwas vereinfachen:

```suggestion
			if (!identifier || !authUser || !authUser.organizationIdentifier) {
```

Reply:
ist raus.